test.py

```diff
@@ -7,8 +7,7 @@
 from pathlib import Path
 import scipy.signal
 import resampy
-import model
-import utils
+from autosynch.umx import model, utils
```

Comment:
I don't know what `autosynch` is

Reply:
Oh crap I copied some extra bits. Sorry.